docs/tutorials/01-intro.md

```diff
@@ -0,0 +1,226 @@
+    buildah inspect $newcontainer
+
+We should probably unmount and commit the image:
+
+     buildah unmount $newcontainer
+     buildah commit $newcontainer fedora-bashecho
+     buildah images
+     
+And you can see there is a new image called `fedora-bashecho:latest`. You can inspect the new image using:
+
+    buildah inspect --type=image fedora-bashecho
+
+Later when you want to create a new container or containers from this image, you simply need need to do `buildah from fedora-bashecho`. This will create a duplicate copy of this image for you.
+
+## OCI images built using Buildah are portable
+
+Let's test if this new OCI image is really portable to another OCI technology like docker. First you should install docker and start it. Notice that Docker requires a daemon process (that's quite big) in order to run any client commands. Buildah has no daemon requirement.
+
+    dnf -y install docker
+    systemctl start docker
+    
+Let's copy that image from where containers/storage stores it to where the docker daemon stores its images, so that we can run it using docker. We can achieve this using `buildah push`. This copies the image to docker's repository area which is located under `/var/lib/docker`. Docker's repository is managed by the docker daemon. This needs to be explicitly stated by telling Buildah to push to the docker repository protocol using `docker-daemon:`.
+
+    buildah push fedora-bashecho docker-daemon:fedora-bashecho:latest
+
+Under the covers, the containers/image library calls into the containers/storage library to read the image's contents, and sends them to the local docker daemon. This can take a little while. And usually you won't need to do this. If you're using `buildah` you are probably not using docker. This is just for demo purposes. Let's try it:
+
+    docker run fedora-bashecho 
+    This is a new container from ipbabble [ 0 ]
+    This is a new container from ipbabble [ 1 ]
+    This is a new container from ipbabble [ 2 ]
+    This is a new container from ipbabble [ 3 ]
+    This is a new container from ipbabble [ 4 ]
+    This is a new container from ipbabble [ 5 ]
+    This is a new container from ipbabble [ 6 ]
+    This is a new container from ipbabble [ 7 ]
+    This is a new container from ipbabble [ 8 ]
+    This is a new container from ipbabble [ 9 ]
+    
+OCI container images built with `buildah` are completely standard as expected. So now it might be time to run:
+
+    dnf -y remove docker
+
+## Using Dockerfiles with Buildah
+
+What if you have been using docker for a while and have some existing Dockerfiles. Not a problem. Buildah can build images using a Dockerfile. The `build-using-dockerfile`, or `bud` for short, takes a Dockefile as input and produces an OCI image.
+
+Find one of your Dockerfiles or create a file called Dockerfile. Use the following example or some variation if you'd like:
+
+    # Base on the Fedora
+    FROM fedora:latest
+    MAINTAINER W Henry email ipbabble@gmail.com
```

Comment:
Just double checking.  Are you happy to have your e-mail exposed once published?